README.md

```diff
@@ -109,8 +110,88 @@ MIT/Expat
 # Contributing
 
 See [CONTRIBUTING.md](CONTRIBUTING.md) if you are interested in internals or
-contributing to the project.
-
+contributing to the project. The original repository provided a [.zenodo.json](.zenodo.json)
+file, and we generate a [.contributors file]() from that via:
+
+```bash
+pip install tributors
+tributors --version
+0.0.15
+```
+
+If you want to have Orcid identifiers, you need to export the following environment
+variables. It also helps to have a GitHub token to increase API limits:
+
+```bash
+export ORCID_ID=xxxxxxxxxx
+export ORCID_SECRET=xxxxxxxxxxxxxxxxxx
+export ORCID_TOKEN=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+export ORCID_REFRESH_TOKEN=xxxxxxxxxxxxxxxxxxxxxxxxx
+export GITHUB_TOKEN=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+```
+
+Instructions for these environment variables can be found [here](https://con.github.io/tributors/docs/getting-started#2-environment). 
+Then update zenodo:
+
+```bash
+tributors update  zenodo
+INFO:    zenodo:Updating .zenodo.json
+INFO:    zenodo:Updating .tributors cache from .zenodo.json
+WARNING:tributors:zenodo does not support updating from names.
+```
+
+In the case that there is more than one orcid found for a user, you will be given a list
+to check. Others will be updated in the file. You can then curate the file as you see fit.
+We next want to add the .allcontributors file:
+
+```bash
+$ tributors init allcontrib
+INFO:allcontrib:Generating .all-contributorsrc for datalad/datalad
+$ tributors update allcontrib
+INFO:allcontrib:Updating .all-contributorsrc
+INFO:allcontrib:Updating .tributors cache from .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor glalteva in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor adswa in .all-contributorsrc
```

Comment:
probably just collapse (remove, and replace with `...`) all but few leading example output lines

Reply:
I almost did that, will do!